Model/Api/Subscriber.php

```diff
@@ -87,19 +131,33 @@ public function updateCustomerCart($customer, $cart)
             'automation'          => 'reset'
         ];
 
-        $customerFields = $this->fieldsBuilder->buildCustomerFields($customer);
-        $cartFields = $this->fieldsBuilder->buildCartFields($quote);
-        $subscriber['fields'] = array_merge($customerFields, $cartFields);
+         $data = $this->helper->extractValues([
+             'cart' => $quote,
+             'cart.products' => $this->helper->getQuoteProducts($quote),
+             'cart.product_categories' => $this->helper->getProductCategories($quote),
+             'customer' => $customer
+             ], $this->helper->getMetaFields());
+         $fields = $this->makeFields($data);
+        $subscriber['fields'] = $fields;
+
+        // $customerFields = $this->fieldsBuilder->buildCustomerFields($customer);
```

Comment:
Is there any reason why it is commented out? Does it mean that request data will not include such field as Subscriber.Source, Subscriber.Firstname etc? I think we should pass them with all orders, as far as Subscriber.Source helps customers to identify where subscriber came from :)